src/zope/password/tests/test_password.py

```diff
@@ -26,11 +37,90 @@
      r"\1"),
     ])
 
+if bcrypt is not None:
```

Comment:
Assuming that the class being tested is defined unconditionally, change this to use deocrators:

- `@unittest.skipIf(bcrypt is None, "bcrypt is not installed")`
- `@unittest.skipIf(bcrypt is not None, "bcrypt is installed")`

E.g., to test the `__init__` I suggested above:

```python
    @unittest.skipIf(bcrypt is not None, "bcrypt is installed")
    def test_ctor_bcrypt_missing(self):
        with self.assertRaises(ValueError) as exc:
            self._makeOne()
        self.assertIn('bcrypt is not installed', str(exc.exception))
```